src/components/signature/Signature.js

```diff
@@ -79,9 +79,10 @@ export class SignatureComponent extends BaseComponent {
   }
 
   build() {
-    if (this.viewOnly) {
-      return this.viewOnlyBuild();
-    }
+    // no need to build a view only (which will display Yes/No) since we need to render the captured signuature instead.
+    // if (this.viewOnly) {
```

Comment:
Why did you comment this out?

Reply:
I see you added some code below that handles viewOnly.  You should just delete this code and not comment it out. We have GIT to keep track of changes.

Reply:
comment has been removed..